<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftFnWin</class>
 <widget class="QDockWidget" name="LeftFnWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>function</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="m_pushButtonExportCpp">
      <property name="text">
       <string>ExportCpp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_pushButtonExportCS">
      <property name="text">
       <string>ExportCS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btnCopyFile">
      <property name="text">
       <string>Copy File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btnConvFile">
      <property name="text">
       <string>Conv File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
